vault/resource_config_ui_custom_message.go

```diff
@@ -146,7 +150,9 @@ func configUICustomMessageRead(ctx context.Context, d *schema.ResourceData, meta
 	}
 
 	if secret == nil || secret.Data == nil {
-		return diag.Errorf("response from Vault server is empty")
+		log.Printf("response from Vault server is empty")
```

Comment:
We need to prepend the log with a level. I would recommend either DEBUG or WARN:
```suggestion
		log.Printf("[DEBUG] response from Vault server is empty, removing from state", vaultPath)
```